Orange/base.py

```diff
@@ -209,12 +209,16 @@ class Model(Reprable):
     Probs = 1
     ValueProbs = 2
 
-    def __init__(self, domain=None):
+    def __init__(self, domain=None, original_domain=None):
```

Comment:
When would `original_domain` be different than domain? Isn't the point of `domain` to specify the original domain in which the model was built? (I know that I was among the people editing this stuff, but I can not remember it now)

Reply:
Ok, now I get it: domain before preprocessing. :)